kudosaurus.go

```diff
@@ -0,0 +1,52 @@
+package kudosaurus
+
+// Kudo - kudo entity
+type Kudo struct {
+	ID          string
+	TeamID      string
+	Text        string
+	PersonFrom  Person
+	PersonTo    Person
+	IsPublished bool
+}
+
+// Person - person entity
+type Person struct {
+	ID         string
+	Name       string
+	Kudos      []Kudo
+	KudosGiven []Kudo
+	KudosLeft  int
+	IsAdmin    bool
+}
+
+// Team - team entity
+type Team struct {
+	ID       string
+	Name     string
+	People   []Person
+	Settings Settings
+}
+
+// Settings - runtime settings for a team
+type Settings struct {
+	TeamID          string
+	TargetChannelID string
+}
+
+// Store - stores data
+type Store interface {
+	PutPerson(id string, name string, teamID string, teamName string) (Person, error)
+	PutKudo(teamID string, idFrom string, idTo string, text string)
```

Comment:
You do not have to write a type each time if consecutive variables are of the same type.
In this case

`PutKudo(teamID, idFrom, idTo, text string)`

would be valid

Reply:
Oooh. Handy.